tests/conftest.py

```diff
@@ -1,7 +1,13 @@
 import asyncio
 import os
+import sys
 
 from twisted.internet import asyncioreactor
 
 os.environ['HATHOR_CONFIG_FILE'] = os.environ.get('HATHOR_TEST_CONFIG_FILE', 'hathor.conf.unittests')
+
+if sys.platform == 'win32':
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+    os.environ['HATHOR_TEST_MEMORY_STORAGE'] = 'true'
```

Comment:
Why only for Windows?

Reply:
Because python-rocksdb does not install on Windows. I considered making this based on whether rocksdb is available or not, but it would have this downside: if rocksdb is accidentally not installed correctly, the tests would fully run, but not cover the parts that we wanted.

Reply:
Got it. Thanks for the explanation.